src/System.Security.Cryptography.Cng/src/System.Security.Cryptography.Cng.csproj

```diff
@@ -8,14 +8,14 @@
     <AllowUnsafeBlocks>true</AllowUnsafeBlocks>
     <IsPartialFacadeAssembly Condition="'$(TargetGroup)' == 'netfx'">true</IsPartialFacadeAssembly>
     <ResourcesSourceOutputDirectory Condition="'$(IsPartialFacadeAssembly)' == 'true'">None</ResourcesSourceOutputDirectory>
-    <GeneratePlatformNotSupportedAssembly Condition="'$(TargetsUnix)' == 'true'">true</GeneratePlatformNotSupportedAssembly>
+    <GeneratePlatformNotSupportedAssembly Condition="'$(TargetGroup)' == 'netstandard'">true</GeneratePlatformNotSupportedAssembly>
     <GenFacadesIgnoreMissingTypes Condition="'$(TargetGroup)'=='netfx'">true</GenFacadesIgnoreMissingTypes>
     <DefineConstants Condition="'$(TargetGroup)' == 'uap'">$(DefineConstants);uap</DefineConstants>
   </PropertyGroup>
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netfx-Windows_NT-Debug|AnyCPU'" />
   <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netfx-Windows_NT-Release|AnyCPU'" />
-  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netcoreapp-Unix-Debug|AnyCPU'" />
-  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netcoreapp-Unix-Release|AnyCPU'" />
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netstandard-Debug|AnyCPU'" />
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netstandard-Release|AnyCPU'" />
```

Comment:
>The types in here had changes in all of 461, 462, 47; so it might need more words here.

The API's had changes? Are those changes present only in desktop, or are they reflected in netcore as well? That would complicate things a bit.

> If all you need is net461 to expose all the API in the contract this is fine.

That is my understanding, yes. 

Here's my line of thinking: this assembly was a part of netstandard20 before being removed recently. net461 supports netstandard20. So, since as a part of the meta-package the net461 binary would be the latest assembly version, the same should be true for the binary in a standalone package.

old package (4.3.0):
net46  -  4.0.0.0
net461 - 4.1.0.0
net463 - 4.2.1.0 (latest)

new package (4.4.0*):
net46   - 4.0.0.0
net461 - 4.3.0.0 (latest)

If net461 can't support the latest assembly (4.3.0.0) for whatever reason, then we need to instead harvest it and make the latest assembly align against a different netfx version. This would have the side effect of net461 (which should support netstandard2.0) having a lower assembly version than the netstandard2.0 refs, which seems troublesome.